modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -261,86 +229,126 @@ class NewProfileIndex extends React.Component {
       site =
         <SelectElement
           name = "site"
-          label = "Site"
+          label = {this.props.t('Site*', {ns: 'new_profile'})}
```

Comment:
Site should come from the loris namespace too?

Also not sure why you have a `*` added that isn't there in the original.